backend/app/controllers/spree/admin/base_controller.rb

```diff
@@ -3,6 +3,8 @@
 module Spree
   module Admin
     class BaseController < Spree::BaseController
+      include Spree::Admin::Breadcrumbs
+
```

Comment:
Could we add the `before_action :set_breadcumbs` here and add an empty method that gets called if the child class does not implement `set_breadcrumbs`? We might even have a "defaut breadcrumb" (build from the resource) for controllers. We could remove a lot of code then.